Datasources: Add user_agent header customization for outgoing HTTP requests (#63769)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/_index.md

Limits the number of rows that Grafana will process from SQL (relational) data sources. Default is `1000000`.

### user_agent

> Sets a custom value for `User-Agent` header for outgoing data proxy requests. If empty, default value will be `Grafana/<BuildVersion>` (e.g. `Grafana/9.0.0`).

Reviewer: ```suggestion
Sets a custom value for the `User-Agent` header for outgoing data proxy requests. If empty, the default value is `Grafana/<BuildVersion>` (for example `Grafana/9.0.0`).
```

Author: Thanks for the suggestion! Addressed.
